### ISSUE TITLE
text_processing2의 두번째 문제에 관한 질문입니다.

### ISSUE THREAD

**Issue by bumjin337** [TEAMLAB-Lecture/python-101-for-data-science-by-pknu#17]
def to_camel_case(underscore_str):
  
    a = input_string.replace('_',' ').strip()

    camelcase_str = ""
    blank_counter = 0
    
    if a.count(' ') == 0 :
        if a.isupper() == True :
            camelcase_str = a.lower()
        else :
            camelcase_str = a
    else :
        for i in a :
            if i.count(' ') == 1 :
                blank_counter += 1
            else :
                if blank_counter == 0 :
                    camelcase_str = camelcase_str + i.lower()
                else :
                    camelcase_str = camelcase_str + i.capitalize()
                    blank_counter = 0

    return camelcase_str

'alreadyCamel'도 잘 나오고 '_______'도 빈 문자열로 반환 되는데 어디서 틀린 건지 잘 모르겠습니다.

**Comment by blissray** [comment]
아무것도 안들어가면 어떻게나오나요?

"" 이런식으로 넣으면

**Comment by bumjin337** [comment]
그냥 빈 문자열로 출력됩니다.

**Comment by blissray** [comment]
예 그 문자열에서 틀리고 있습니다 "" 이렇게 나와야 하는데 "       " 이렇게 나오네요 len으로 글자의 개수를 찍어보세요.

**Comment by bumjin337** [comment]
저 코드대로 len을 써보면 글자의 개수가 0개로 나오는데 원래라면 2개로 나와야 하는 건가요??
